### ISSUE TITLE
🔗 ♿ 🐞 Bug Report: Accordion and Toggle Widgets Titles Empty Link May Cause Problems for Good Bots (Merged with #14371) - [ED-8824]

### ISSUE THREAD

**Issue by Aikobot** [elementor/elementor#11611]
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description
This may be more of a compatibility issue than technically a "bug", so my apologies if I am reporting in the wrong place.

These two widgets standardly create an empty link for each accordion or toggle entry like this: `<a href=""><a/>`. I believe the purpose of this is perhaps just to facilitate hover styling for the accordion entry title. While it may behave as intended it doesn't seem semantically correct, and more importantly might contribute to a very undesirable issue. 

We've had a large problem resulting in thousands of 404 errors in our Moz campaign and CPU Limit overage with our hosting. As the Moz Help team describes, their crawler for campaign audits - Rogerbot will append the relative link to the end of the existing link it is on. Thus a page with a relative link ending in say "/products/" generates many 404 instances like "/products/products/ products/" and so on. So they recommend not using relative urls in the source code to prevent generating these 404s. While at this point I can't say for certain, I think that Rogerbot essentially treats this empty link the same as it does with relative links. And while I would prefer that Rogerbot didn't handle relative or empty links this way, it seems like an improvement in the Elementor code could help with compatibility between non-malicious bots and Elementor, without loss of function for Elementor users.

If my assessment of the cause of this issue is correct, I'm suggesting that a simple change in the code for Accordions and Toggles could solve the issue . Can/should the empty link code be removed/replaced with another method for hover styling such as that used by various other third-party accordion widget addons?

## Isolating the problem

- [X ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X ] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
For this issue I think system information is irrelevant. But I will happily supply it you deem it relevant.
```
</details>

Thank you!!!

**Comment by StachRedeker** [comment]
This issue provides a solution:
https://github.com/elementor/elementor/issues/8943

And you can combine it with this tutorial to implement the script:
https://stackoverflow.com/questions/11159860/how-do-i-add-a-simple-jquery-script-to-wordpress

Worked for me.

**Comment by mikdeveloper007** [comment]
@StachRedeker I added that in the footer of the website and it worked for me

**Comment by nicholaszein** [comment]
Hello, everyone!

Thank you for your feedback! 🙏 

### 🔗 🔃 🐞 This issue is being merged with #14371.

Which is on the same subject. 

For this reason, I'm closing this thread so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

Kind regards

**Comment by nicholaszein** [comment]
Hello!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.14.0! 🥳 

Now Accordions are completely a11y compatible! 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
